airflow/migrations/versions/0032_3_0_0_rename_execution_date_to_logical_date_and_nullable.py

```diff
@@ -61,9 +66,11 @@ def downgrade():
             "logical_date",
             new_column_name="execution_date",
             existing_type=TIMESTAMP(timezone=True),
-            existing_nullable=False,
+            nullable=False,
```

Comment:
in the downgrade, we would need to do something about the nulls. either delete the runs, or move them to an archive table, or populate them with 1900-01-01 or something.  @jedcunningham ?

Reply:
hmm, depends how cheeky I'm feeling haha. The downside of populating a old fake value is the need for it to be unique, seems like a pain.

Reply:
Maybe move them to a separate table? We did that for some records when we added many unique constraints on TI back in 2.2 or something.

Reply:
I created #46500 for this.